src/Services/ShipmentNotificationService.php

```diff
@@ -495,6 +500,18 @@ public function prepareRequestDto(Order $order): RequestDTO
         $orderTrackingNumbers = $this->orderRepositoryContract->getPackageNumbers($orderId);
         $requestDto->setPackageCount(count($orderShippingPackages) > 0 ? count($orderShippingPackages) : 1);
 
+        $this->loggerContract->debug(
+          TranslationHelper::getLoggerKey(self::LOG_KEY_SHIPPING_ON_OWN_ACCOUNT),
```

Comment:
It looks like this log is now going to appear twice, with slightly different info, at different levels?